### ISSUE TITLE
Port the remaining functions of Shared Playlists and fix its current bugs

### ISSUE THREAD

**Issue by Reddnek** [Reddnek/syncplay-android#19]
Syncplay Android has the following known bugs when it comes to Shared Playlists:
- When a user joins a room that has users playing a file other than the first on top of the list, the user loads the first video anyway. This is  an inconvenience and a bug to be fixed.
- During scrolling syncplay's Shared Playlists, some files may appear "playing" while they are not (a bug in the RecyclerView adapter)

Syncplay Desktop has the following features that should be ported to Syncplay Android:
- Ability to set media directories ***inside the room*** the room.
- Ability to set trusted domains (When URL support is added)
- Saving and loading a playlist to/from a local file.
- Shuffle controls
- Basically every other option when you right-click in the shared playlists section in Syncplay Desktop.

**Comment by Reddnek** [comment]
Shared Playlists are fully supported as of version [0.10.0](https://github.com/Reddnek/syncplay-android/releases/tag/v0.10.0).
